src/app/components/Company.js

```diff
@@ -30,6 +30,7 @@ class Company extends React.Component {
 
   render() {
     const { companyData, checklist } = this.props;
+    const scorePercentage = Math.round((companyData.score / 13) * 100);
```

Comment:
Do we have access to the `frontendData` in this component? Maybe we could divide the score by the length of the frontendData....I am not sure if that would make sense though

Reply:
Good idea! In case we ever change the number of checklist items